hummingbot/connector/exchange/bittrex/bittrex_exchange.pyx

```diff
@@ -698,9 +698,9 @@ cdef class BittrexExchange(ExchangeBase):
                 "type": "LIMIT",
                 "quantity": f"{amount:f}",
                 "limit": f"{price:f}",
-                "timeInForce": "GOOD_TIL_CANCELLED"
-                # Available options [GOOD_TIL_CANCELLED, IMMEDIATE_OR_CANCEL,
-                # FILL_OR_KILL, POST_ONLY_GOOD_TIL_CANCELLED]
+                "timeInForce": "GOOD_TIL_CANCELED"
+                # Available options [GOOD_TIL_CANCELED, IMMEDIATE_OR_CANCEL,
+                # FILL_OR_KILL, POST_ONLY_GOOD_TIL_CANCELED]
```

Comment:
This can't be changed. The correct value according to Bittrex API documentation is `GOOD_TIL_CANCELLED`